lxd/instance/drivers/driver_lxc.go

```diff
@@ -5566,6 +5566,8 @@ func (d *lxc) Console(protocol string) (*os.File, chan error, error) {
 		return nil, nil, err
 	}
 
+	d.lifecycle(lifecycle.CONSOLE, nil)
```

Comment:
For console (especially in the VM case), I think the context should include the type of console (`console` in this case, either `console` or `vga` in the VM case).